src/components/cesium_map/cesium_UI.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * This is a uncontrolled react component
+ * See link:
+ * https://goshacmd.com/controlled-vs-uncontrolled-inputs-react/
+ */
+
+import React from "react";
+import * as Cesium from "cesium";
+import {
+  SpatialView,
+  ISamplesSpatial,
+  PointStreamDatasource,
+  PointStreamPrimitiveCollection
+} from "./api/spatial";
+import { ISamplesAPI } from "./api/server";
+
+// Defined ceisum access token
+// Current one is Dave's token
```

Comment:
If we have documentation on how this is generated, it would be nice to include a link to it here.

Reply:
This is my fault. I did add a link yesterday. But I reverted this file by accident.

Reply:
No problem, thank you for adding it